scripts/testing/install_dependencies.sh

```diff
@@ -28,7 +28,7 @@
 
 set -eu
 
-dnf install $@ rpm-build /usr/bin/xargs
+dnf install $@ make rpm-build /usr/bin/xargs npm libvirt virt-install
```

Comment:
Isn't libvirt direct dependency of virt-install? 

If not, libvirt-daemon-kvm and libvirt-client are sufficient. Libvirt is a metapackage and installs a lot of uneeded things.

Reply:
I did not know about the libvirt-client, in that case yes, virt-install has a dependency on libvirt-client.

Reply:
Fixed.